docusaurus.config.js

```diff
@@ -213,7 +213,7 @@ const config = {
           },
           {
             from: "/Explorer-Backend-Starter-Guide.html",
-            to: "/full-node/explorer-backend",
```

Comment:
There should also be a new redirect rule from `/full-node/explorer-backend` to the new endpoint `/infrastructure/explorer-backen` otherwise we'll have broken links.